lesson7-questions.txt

```diff
@@ -32,7 +43,7 @@ that if you have a @subscription (an instance of Subscription), you have a coupl
 used to refer to associated database methods.  What are those methods?  (Actually there are a few,
 but there are two in particular you should know.)
 
-
+it takes from the user and forum methods; those take from posts, subscriptions
```

Comment:
just clarifying -- `@subscription` now has two methods, `user` and `forum`. because it belongs to one user and one forum, it will pull up the associated `user` and `forum`. these in turn bring up instances of User and Forum!